packages/components/app/styles/components/dropdown.scss

```diff
@@ -169,7 +170,22 @@ $hds-dropdown-toggle-border-width: 1px;
     isolation: isolate; // used to create a new stacking context (needed to have the pseudo element below text/icon but not the parent container)
     min-height: 36px;
 
-    a {
+    // need to reset a few extra things to make the button visually appear the same as the links
+    // also CR if you want to re-factor, that's fine; just trying to make this work in this first pass
```

Comment:
@MelSumner I can't refactor until we remove the nesting of classes (needs to be done in a separate PR)